modules/rosa-deleting-cluster.adoc

```diff
@@ -32,6 +46,34 @@ $ rosa init --delete-stack
 ----
 endif::sts[]
 
+ifdef::sts[]
+.  Delete the OpenID Connect (OIDC) provider that the cluster Operators use to authenticate:
++
+[source,terminal]
+----
+$ rosa delete oidc-provider -c <cluster_id> --mode auto --yes <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster.
+
+. Delete the cluster-specific Operator IAM roles:
++
+[source,terminal]
+----
+$ rosa delete operator-roles -c <cluster_id> --mode auto --yes <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster.
+
+. Delete the account-wide roles:
++
+[source,terminal]
+----
+$ rosa delete account-roles --prefix <prefix> --mode auto --yes <1>
+----
+<1> Replace `<prefix>` with the prefix of the account roles that are being deleted, for example `ManagedOpenShift`.
+
+. Delete the account-wide inline and Operator policies by using the AWS IAM Console. For detailed steps, see the _Deleting the AWS resources by using the AWS IAM Console_ section.
```

Comment:
Recommend recasting to be more direct as: "Use the AWS IAM Console to delete the account-wide inline and Operator policies."

Reply:
Thank you for this suggestion. Resolved.